### PR TITLE
fixed text not clearly visible in dark mode

### PR DESCRIPTION
**Pull request by sourav-sharma7**
Signed-off-by: sourav-sharma7 <souravsharma1321@gmail.com>

**Description**
text colour changed for dark mode in service-mesh-management/meshery/integrations/windows 
initial 
Screenshot from 2022-08-19 19-00-14
FInal 
Screenshot from 2022-08-20 18-08-31

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Meshery/Meshery-integrations/Individual-Integrations/howItWork.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import SMPImage from "../../images/meshery_multi_mesh.png";
 
 const HowIntegrationWorksWrapper = styled.section`
 
-  background: #F7F7F7;
+  background: ${props => props.theme.DarkTheme};
```
Comment (Nikhil-Ladha) [inline src/sections/Meshery/Meshery-integrations/Individual-Integrations/howItWork.js:10]: You need to switch the color based on this value. Not, just return the value.

**Comment by l5io** [comment]
🚀 Preview for commit 4a47cdbbab3f3a51a6b8edcaf603f2c3f30229b1 at: https://6300db9e44ba2e42fad4fec9--layer5.netlify.app

File: src/sections/Meshery/Meshery-integrations/Individual-Integrations/howItWork.js (hunk 1 above)
Comment (sourav-sharma7) [reply src/sections/Meshery/Meshery-integrations/Individual-Integrations/howItWork.js:10]: Can you please elaborate. The background color is getting changed based on the theme selection.

Comment (Nikhil-Ladha) [reply src/sections/Meshery/Meshery-integrations/Individual-Integrations/howItWork.js:10]: Well, the section color is not changing but the background color of the complete HTML is and the same is reflecting in the section.
Update the code to `background-color: ${props => props.theme.DarkTheme ? #101010 : #F7F7F7};`

Comment (Nikhil-Ladha) [reply src/sections/Meshery/Meshery-integrations/Individual-Integrations/howItWork.js:10]: Also, while you are making the change can you please apply a similar update to the link color of `See Documentation` text?
Use `colorr: ${props => props.theme.text };` It should do.

Comment (sourav-sharma7) [reply src/sections/Meshery/Meshery-integrations/Individual-Integrations/howItWork.js]: Thanks for the detailed explanation. I got it now. Can you please review new changes.

File: src/sections/Meshery/Meshery-integrations/Individual-Integrations/individual-integrations.style.js [hunk 2]
```diff
@@ -83,7 +83,7 @@ export const IntegrationPageWrapper = styled.section`
 				gap: 0.5rem;
 				align-items: center;
 				a {
-					color: black;
+					color: ${props => props.theme.primaryColor};
```
Comment (Nikhil-Ladha) [inline src/sections/Meshery/Meshery-integrations/Individual-Integrations/individual-integrations.style.js:86]:
```suggestion
					color: ${props => props.theme.text};
```

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**